data/Makefile.am

```diff
@@ -33,8 +36,16 @@ desktop_DATA =						\
 	setup-shell.desktop				\
 	$(NULL)
 
+demovideodir = $(datadir)/eos-demo-mode
+demovideo_DATA = \
+	eos-demo-video.ogv \
+	eos-demo-video.desktop \
```

Comment:
I'd rather you installed this file in `/etc/xdg/autostart` on the system in general, and use an `AutostartCondition` rule in the desktop file to only launch it for demo sessions.